Tests/ContainerTest.php

```diff
@@ -38,6 +45,12 @@ public static function setUpBeforeClass(): void
      */
     public function testContainer(): void
     {
+        if (interface_exists(EntityManagerInterface::class)) {
+            return;
```

Comment:
You can't copy paste this block of code from previous method, it needs to be adjusted. Currently rest of the body is skipped for both conditions.

Reply:
I don't understand your comment, or why this block was in `setUpBeforeClass()` in the first place: there is only one method, the one I am adding should not be skipped (and isn't)

Reply:
```php
         self::markTestSkipped('This test requires ORM');
```
is early return, same as `return;` I am commenting here. So if EntityManagerInterface doesn't exist, return. Otherwise, skip (=return). That's what code in this PR is currently doing (nothing).

It was fine to use it like that in `setUpBeforeClass()` because if that method just returns null without doing null, testcase would still be executed.

Reply:
Oh right, how did I not spot this, that's embarassing :sweat_smile: